src/ResourceManagement/HDInsight/HDInsight.Tests/Helpers/GetClusterSpecHelpers.cs

```diff
@@ -134,6 +141,136 @@ public static ClusterCreateParametersExtended GetIaasClusterSpec()
             return cluster;
         }
 
+        public static ClusterCreateParametersExtended GetIaasAdJoinedClusterSpec()
```

Comment:
can you add a test for ClusterCreateParameters please? That's the more common scenario.